### DISCUSSION TITLE
Re-Ordering Selection Menu

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#16106] [Feature requests] [answered]
I am not gonna lie, when I say that the current order of everything isn't very coherent.
Perhaps, since so much is changing this update it'd also be a good time to re-order the menu.
Here is my suggestion of keeping everything together as much as possible.
![image](https://user-images.githubusercontent.com/1516893/144676419-b635ee89-4a3b-4055-b638-8b8385ef1463.png)

If you wish to edit this easily here's a PSD I made where I split up the icons :3 
[Selection PSD.zip](https://github.com/OpenRCT2/OpenRCT2/files/7652715/Selection.PSD.zip)

Two public polls on discord: 
https://discord.com/channels/601789725090643968/601794896189194269/916455684135075900 (Marcel's Server)
https://discord.com/channels/493142638728314890/493167642853638164/916455183955951676 (Deurklink's Server)
https://discord.com/channels/234774211330048012/234774211330048012/916464114937782293 (New Element)
https://discord.com/channels/356546567391674369/450316748734988289/916464196516970497 (RC & Friends)

**Comment by oli414** [comment]
You should just be able to drag the PSD into the text file to upload it I think.

This change is going to be a bit controversial but I think it would be better this way.

I'd suggest putting terrain and water between the banners and park entrances tough. Just to keep it a bit more similar to what people are used to right now (Doesn't impact the logical order of things imo)

**Reply by karst** [reply]
I tried dragging the PSD into the text file but it didnt let me sadly

**Reply by karst** [reply]
![idea2](https://user-images.githubusercontent.com/1516893/144680713-092c7f9f-e3a0-4745-9cdb-a8155c2b791a.png)
So your suggestion would be this, correct?

**Reply by oli414** [reply]
Yeah

If you zip the PSD first it should let you

**Reply by karst** [reply]
Ah fixed that!

**Comment by karst** [comment]
Current results from the polls.

Original: 5
Order 1: 12
Order 2: 19

There was also a suggestion in New Element if it would be possible to make the order customizable per person.

**Comment by karst** [comment, marked as answer]
Implemented in #16952
